### PR TITLE
remove backward compat for alpha-zng

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

**Comment by nwt** [comment]
I think the failure at https://github.com/brimsec/zq/pull/2158/checks?check_run_id=1922146674 is flakiness related to #2145.

File: ppl/zqd/storage/filestore/filestore.go [hunk 1]
```diff
@@ -208,16 +201,13 @@ func (s *Storage) readInfoFile() error {
...
 	s.span = nano.NewSpanTs(inf.MinTime, inf.MaxTime)
-	s.alphaMigrated = inf.AlphaMigrated
 	return nil
 }
 
 func (s *Storage) syncInfoFile() error {
 	// If we are updating the info file, we've either written data, or are clearing
 	// the info file so that we can write data later. In either case, we'd be using
 	// the post-alpha zng format.
-	s.alphaMigrated = true
```
Comment (alfred-landrum) [inline ppl/zqd/storage/filestore/filestore.go:219]: You can remove the comment above this line as well.

**Comment by nwt** [comment]
- [ ] Remove `.gitattributes`
- [ ] Remove `"azng" from `Extension` in zio/zio.go

File: ppl/zqd/storage/filestore/filestore.go [hunk 2]
```diff
@@ -230,39 +220,11 @@ func (s *Storage) syncInfoFile() error {
 	info := info{
 		MinTime:       s.span.Ts,
 		MaxTime:       s.span.End(),
-		AlphaMigrated: s.alphaMigrated,
+		AlphaMigrated: true,
```
Comment (alfred-landrum) [inline ppl/zqd/storage/filestore/filestore.go:223]: You can remove the `AlphaMigrated` field from the `info` type definition, as there won't be anything paying attention to it.

Comment (mccanne) [reply ppl/zqd/storage/filestore/filestore.go:223]: I wasn't sure if the app would do something...

**Review by alfred-landrum — APPROVED** [review]

**Review by nwt — APPROVED** [review]
:shipit:!

**Pull request merged**